aten/src/ATen/native/Convolution.cpp

```diff
@@ -972,7 +981,7 @@ static Tensor convolution_same(
     SymIntArrayRef stride, SymIntArrayRef dilation, c10::SymInt groups) {
 
   auto k = weight.dim();
-  TORCH_CHECK(k > 2, "weight should have at least three dimensions");
```

Comment:
why this change

Reply:
We don't need to remove this check. I've updated it. Thanks for pointing it out, @lezcano!